routes/api/user.js

```diff
@@ -3,46 +3,46 @@ const usersController = require('../../controllers/usersController');
 
 module.exports = (app) =>{
 
-    //sign up
-    app.post('/user',usersController.findOrCreate);
-        
+  //sign up
+  app.post('/user',usersController.findOrCreate);
 
+  app.post(
+    '/user/login',
+    (req, res, next) =>{
+      console.log('routes/user.js, login, req.body: ');
```

Comment:
do we need this?

Reply:
it a passport login, but I can remove the console.log.